ewkcoffea/modules/selection_wwz.py

```diff
@@ -68,6 +68,7 @@
             "Ele32_WPTight_Gsf_L1DoubleEG",
             "Ele35_WPTight_Gsf",
             "Ele115_CaloIdVT_GsfTrkIdT",
+            "DoublePhoton70",
```

Comment:
Was this intended to be added? Does it make any impact?

Reply:
Technically, Guillermo is using this. I can perform a quick comparison with it on/off in file-ton to see if there is any effect.

Reply:
Was there any effect?

Reply:
very negligible change (5 orders of magnitude difference)

Reply:
For di lepton for 4l?

Reply:
dilepton

Reply:
Ok, I'm good with removing it, as discussed on matter most.